perl.h

```diff
@@ -179,7 +179,14 @@ Otherwise ends a section of code already begun by a C<L</START_EXTERN_C>>.
 /* Fallback definitions in case we don't have definitions from config.h.
    This should only matter for systems that don't use Configure and
    haven't been modified to define PERL_STATIC_INLINE yet.
+
+=for apidoc CmU|void|PERL_STATIC_INLINE|token
```

Comment:
I don't think this should be "C" core only.

The name as used is prefixed and has to be visible to other code for `inline.h` to work, so I think it's safe for it to be documented as API.